### DISCUSSION TITLE
How do you enable JIT

### DISCUSSION THREAD

**Discussion by SuperiorSpiderman** [SideStore/SideStore#276] [Q&A] [answered]
I am a new user to Sidestore, and recently installed DolphinIOS. It says there is an in-app JIT enabler, but I don't know how to do it. You think any of you guys coud help me? Thanks!

**Comment by Spidy123222** [comment, marked as answer]
> I am a new user to Sidestore, and recently installed DolphinIOS. It says there is an in-app JIT enabler, but I don't know how to do it. You think any of you guys coud help me? Thanks!

You go in "My Apps" hold down on app and click enable jit. This is the same for AltStore.

**Comment by IYY66** [comment]
加油

**Comment by Scvairy** [comment]
Can it activate JIT for the app installed through AltStore PAL? The app is not listed

**Reply by nythepegasus** [reply]
As far as I know those apps don’t get debug entitlements, so no.

**Reply by Scvairy** [reply]
Oh, as I suspected...
Okay, then, I'll try to sideload utm and enable JIT that way, thanks!
